core/server/master/src/main/java/alluxio/master/meta/DefaultMetaMaster.java

```diff
@@ -623,27 +623,31 @@ public void resetState() {
   public Map<String, Boolean> updateConfiguration(Map<String, String> propertiesMap) {
     Map<String, Boolean> result = new HashMap<>();
     int successCount = 0;
+    if (!Configuration.getBoolean(PropertyKey.CONF_DYNAMIC_UPDATE_ENABLED)) {
+      LOG.warn("{} should be true, Please change Settings",
+          PropertyKey.CONF_DYNAMIC_UPDATE_ENABLED.getName());
```

Comment:
```suggestion
      LOG.warn("Dynamic configuration update disabled but attempted, set {} to true to enable it",
          PropertyKey.CONF_DYNAMIC_UPDATE_ENABLED.getName());
```

Reply:
I think also doing this check at the client side makes more sense in terms of human friendliness and quality of life. You can get a prompt error message telling you this feature is disabled and needs to be enabled before use.

Reply:
If the configurations is different between client-side and master-side, so the client-side `CONF_DYNAMIC_UPDATE_ENABLED ` check would be incorrect?

Reply:
First of all, the client-master config sync is enabled by default, the client will pull the cluster-default config from master first before doing anything.

If the master side config disables the feature (the default), but the client enables it by overwriting this config, then the client will try to do the update anyway. They just cannot see this precise and friendly error message in their terminal, and have to dig into master's log.

If the master enables the feature, but the client somehow explicitly disables the feature, then I'm sure a sane user must have a valid reason not to allow dynamic config update on that particular client. If the client does not explicitly disable it, it will pick up the master's config, and the check won't prevent the client from doing the update.